CHANGELOG.md

```diff
@@ -7,65 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Run yarn lavamoat:auto ([#13697](https://github.com/MetaMask/metamask-extension/pull/13697))
-- Using correct metricsEvent arg ([#13684](https://github.com/MetaMask/metamask-extension/pull/13684))
-- Added Controls and Action ArgType, took out @storybook/addon-knobs  ([#13660](https://github.com/MetaMask/metamask-extension/pull/13660))
-- Add support for Smart Transactions ([#12676](https://github.com/MetaMask/metamask-extension/pull/12676))
-- Add argTypes for color-indicator ([#13659](https://github.com/MetaMask/metamask-extension/pull/13659))
-- Enable Avalanche for Swaps ([#13653](https://github.com/MetaMask/metamask-extension/pull/13653))
-- Prevent Browserify error from being swallowed ([#13647](https://github.com/MetaMask/metamask-extension/pull/13647))
-- Mįgrate truncated definition list stories from addon knobs to use controls. ([#13628](https://github.com/MetaMask/metamask-extension/pull/13628))
-- Fixed Add Recipient Stories and migrated to use control args. ([#13633](https://github.com/MetaMask/metamask-extension/pull/13633))
-- Minimal fix to the TokenSearch story to ensure it compiles and works. ([#13623](https://github.com/MetaMask/metamask-extension/pull/13623))
-- Updated the Callout story to convert knobs and actions to controls / args. ([#13624](https://github.com/MetaMask/metamask-extension/pull/13624))
-- Change "Quote X of Y" to "Fetching quote X of Y" ([#13663](https://github.com/MetaMask/metamask-extension/pull/13663))
-- Sync `master` with `develop` ([#13656](https://github.com/MetaMask/metamask-extension/pull/13656))
-- convert SendGasRow stories with argtypes ([#13554](https://github.com/MetaMask/metamask-extension/pull/13554))
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- Use hardware wallet constants when possible ([#13634](https://github.com/MetaMask/metamask-extension/pull/13634))
-- ConfirmImportToken: Convert to Functional Component; Add storybook controls ([#13594](https://github.com/MetaMask/metamask-extension/pull/13594))
-- Ensure pre-selected account on connect screen is visible in list ([#13621](https://github.com/MetaMask/metamask-extension/pull/13621))
-- Sync `master` with `develop` ([#13648](https://github.com/MetaMask/metamask-extension/pull/13648))
+### Added
+- Swaps: Add support for Smart Transactions on Mainnet and Rinkeby ([#12676](https://github.com/MetaMask/metamask-extension/pull/12676))
+- Swaps: Enable Avalanche ([#13653](https://github.com/MetaMask/metamask-extension/pull/13653))
+- Add "Token details" page ([#13216](https://github.com/MetaMask/metamask-extension/pull/13216))
+- Add icons to sections in settings ([#12920](https://github.com/MetaMask/metamask-extension/pull/12920))
+
+### Changed
+- Swaps: Change language "Quote X of Y" to "Fetching quote X of Y" ([#13663](https://github.com/MetaMask/metamask-extension/pull/13663))
 - Make default icons the same on mobile and extension #13264 ([#13408](https://github.com/MetaMask/metamask-extension/pull/13408))
```

Comment:
Suggest that we change `Make default icons the same on mobile and extension` to "Update preferred account identicon selection UI"

Reply:
Updated